mock/mockIsdialogmote.js

```diff
@@ -32,6 +37,11 @@ function mockSyfomoteadmin(server, erLokal) {
     res.setHeader('Content-Type', 'application/json');
     res.send(referat);
   });
+
+  server.post('/syk/dialogmote/api/v1/arbeidstaker/brev/:uuid/respons', (req, res) => {
```

Comment:
Ser ut som denne er lik den over i samme scope?

- [ ] Fjern

Reply:
Den ene er inne i `if (erLokal)`. Men kan vi egentlig slette hele den `if (erLokal)`-blokken? Har den noen funksjon?

Reply:
Stemmer 😅

Reply:
Ser ut som det kan fjernes ja

Reply:
Det kan fjernes men da må vi være bevisst på endringene for de reflekteres i labs

Reply:
Akkurat nå er de jo nærmest identiske. Men du tenker kanskje at vi må være bevisst på fremtidige endringer, @luantr? At man ikke må pushe endringer i mockdata/endepunkt uten å tenke på at det påvirker labs?

Reply:
Er det vits å ha ulik data fra lokalt og labs?

Reply:
Ja, det var det jeg tenkte på @kaja-nav.

Egentlig ikke :P For øyeblikket så skal ikke labs ha referat, innkallelse etc så det må legges hvis man ønsker å se det lokalt.

Reply:
Nå har jeg fjernet if, så nå er det samme mock data for lokal og labs.